Added mentions of load balancing v2 (#462)
spatialos/UnrealGDK: SpatialGDK/Documentation/content/toolbars.md

You can launch multiple servers at the same time from within the Unreal Editor in [PIE (Unreal documentation)](https://docs.unrealengine.com/en-us/Engine/UI/LevelEditor/InEditorTesting#playineditor) configuration. To configure the number of servers launched, open the **Play** drop-down menu and use the slider `Number of Servers` within the `Multiplayer Options` section.

> To connect multiple servers-workers to SpatialOS, we need to tell SpatialOS how many server-workers we will be connecting. In `<ProjectPath>\spatial\default_launch.json` there is a load balancing section which dictates how many workers will be connected to SpatialOS. 

Reviewer: Does this whole new section assume that the user has switched to LBV2? Shouldn't we mention somewhere that there's a new load balancing system that you have to upgrade to, and you have to upgrade to the new bridge first, etc.?

Author: `default_launch.json` uses LBV2 by default.

Reviewer: Ah right - do we make that clear anywhere in the GDK docs? I think we should - even if it's just a note wherever relevant (which might be multiple places).

> To connect multiple servers-workers to SpatialOS, we need to tell SpatialOS how many server-workers we will be connecting. In `<ProjectPath>\spatial\default_launch.json` there is a load balancing section which dictates how many workers will be connected to SpatialOS. 

Reviewer: ```suggestion
To connect multiple servers-workers to SpatialOS, you need to tell SpatialOS how many server-workers you will be connecting. In `<ProjectPath>\spatial\default_launch.json` there is a load balancing section which dictates how many workers will be connected to SpatialOS. 
```

If you haven't modified your load balancing previously, your load balancing strategy should look like:

```
"load_balancing": {
  "layer_configurations": [
      {
          "layer": "UnrealWorker",
          "rectangle_grid": {
              "cols": 1,
              "rows": 1
          },
          "options": {
            "manual_worker_connection_only": true
        }
      }
  ]
}
```

> This uses the [`rectangle_grid`](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancer-config-2#rectangular-grid-rectangle-grid) strategy with 1 column and 1 row. To connect 2 servers, change this to 1 column and 2 rows (or vice-versa). Read more about the different kinds of load balancing strategies [here](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancer-config-2#load-balancing-with-the-new-runtime)

Reviewer: ```suggestion
This uses the [`rectangle_grid`](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancer-config-2#rectangular-grid-rectangle-grid) strategy with 1 column and 1 row. To connect 2 servers, change this to 1 column and 2 rows (or vice-versa). Read more about the different kinds of load balancing strategies [here](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancer-config-2#load-balancing-with-the-new-runtime).
```

Missed a full stop

> This uses the [`rectangle_grid`](https://docs.improbable.io/reference/13.3/shared/worker-configuration/load-balancer-config-2#rectangular-grid-rectangle-grid) strategy with 1 column and 1 row. To connect 2 servers, this can be changed to 1 column and 2 rows (or vice-versa). Read more about the different kinds of load balancing strategies [here](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancer-config-2#load-balancing-with-the-new-runtime)

Reviewer 2: ```suggestion
This uses the [`rectangle_grid`](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancer-config-2#rectangular-grid-rectangle-grid) strategy with 1 column and 1 row. To connect 2 servers, this can be changed to 1 column and 2 rows (or vice-versa). Read more about the different kinds of load balancing strategies [here](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancer-config-2#load-balancing-with-the-new-runtime)
```

> This uses the [`rectangle_grid`](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancer-config-2#rectangular-grid-rectangle-grid) strategy with 1 column and 1 row. To connect 2 servers, this can be changed to 1 column and 2 rows (or vice-versa). Read more about the different kinds of load balancing strategies [here](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancer-config-2#load-balancing-with-the-new-runtime)

Reviewer: ```suggestion
This uses the [`rectangle_grid`](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancer-config-2#rectangular-grid-rectangle-grid) strategy with 1 column and 1 row. To connect 2 servers, change this to 1 column and 2 rows (or vice-versa). Read more about the different kinds of load balancing strategies [here](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancer-config-2#load-balancing-with-the-new-runtime)
```

Reviewer: "or vice-versa" - does this mean that it doesn't matter whether it's 1 column and 2 rows, or 2 columns and 1 row? (If so, fine - that's what I took it to mean :))
